server/webserver.go

```diff
@@ -717,10 +701,17 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 				if !init {
 					return errors.New("Round not initialized")
 				}
-				err = s.LivepeerNode.Eth.ClaimEarnings(endRound)
+				currRound, err := s.LivepeerNode.Eth.CurrentRound()
 				if err != nil {
 					return err
 				}
+				tx, err := s.LivepeerNode.Eth.ClaimEarnings(currRound)
+				if err != nil {
+					return err
+				}
+				if err := s.LivepeerNode.Eth.CheckTx(tx); err != nil {
```

Comment:
We can shorten this to `return s.LivepeerNode.Eth.CheckTx(tx)`.

Reply:
Good point, fixed c6cb779